tests/processes/ProcessingTest.py

```diff
@@ -566,7 +566,8 @@ def test_yield_ignore_ranges(self):
     def test_loaded_bears_with_error_result(self):
         class BearWithMissingPrerequisites(Bear):
 
-            def __init__(self, section, queue, timeout=0.1, debugger=False):
```

Comment:
Why do these need to be changed?

Is it mandatory for bears `__init__` to have a new `debugger` and `profile` args?
That would be a pretty serious breakage for an enhancement which is suppose to happen transparently to bears.

Why specify defaults at all?  core will require that bears add new parameters and copy those defaults or risk a breakage because the `__init__` is called without specifying values for args.

Reply:
@Makman2  ^^ ping 
Discussion happend here need your suggestion :point_right: : [July 20, 2018 3:50 PM](https://gitter.im/coala/coala?at=5b51b75163cf1636bdde13bd)

Reply:
@Makman2 , lots of small breaking changes over 1.5 years means we loose users in the upgrade.  They can handle one or two minor changes affecting them , but they give up after a while, especially when things break in `__init__`.

Reply:
Makes sense :+1: Then let's use the `section` directly then :+1: CC @RaiVaibhav

Reply:
However, let's merge this first analogous to the `debugger` feature which is already merged, and then push a commit to enforce API compatibility again.

Reply:
:+1: My first job will be to solve the API breaking problem, just after this code gets merge and fix